source/plot/Axis.ooc

```diff
@@ -0,0 +1,170 @@
+use ooc-math
+import svg/Shapes
+import math
+
+Orientation: enum {
+	HORIZONTAL
+	VERTICAL
```

Comment:
Enums are written like class names, so `Horizontal` and `Vertical`, respectively.